build.sbt

```diff
@@ -52,16 +50,6 @@ lazy val scala211 = "2.11.12"
 lazy val scala212 = "2.12.8"
 lazy val scala213 = "2.13.0"
 
-scalaVersionsByJvm in ThisBuild := {
```

Comment:
maybe this was still in use to ensure we don't publish on jdk 11, so we'll have to add a condition to the `ci` command.